<launch>
  
  <param name="lat_r" value="45.477669" />
  <param name="lon_r" value="9.226740" />
  <param name="alt_r" value="169.041000" />
 
   <node pkg="first_project" name="gps_to_odom" type="gps_to_odom">
 </node>
 
 
  <group ns="group_wheel_odom">
 	<node pkg="first_project" name="odom_to_tf" type="odom_to_tf" output="screen">
 		<param name="input_odom" value="/odom" />
 	</node>
 </group>
 
 <group ns="group_gps_odom">
 	<node pkg="first_project" name="odom_to_tf" type="odom_to_tf" output="screen">
 		<param name="input_odom" value="/gps_odom" />
 	</node>
 </group>

</launch>
